packages/cubejs-query-orchestrator/orchestrator/RedisCacheDriver.js

```diff
@@ -1,25 +1,47 @@
-const createRedisClient = require('./RedisFactory');
-
 class RedisCacheDriver {
-  constructor() {
-    this.redisClient = createRedisClient(process.env.REDIS_URL);
+  constructor(pool) {
```

Comment:
Would it be possible for the constructor options to be an object, with a property `pool` instead? This would be helpful in the future, in case we need to add any more parameters to instantiate a Redis driver

Reply:
@hassankhan Yeah. Great idea! I think we should provide options object to `RedisPool` instead of arguments.

Reply:
https://github.com/cube-js/cube.js/pull/468